### PR TITLE
Changed dropdown volunteer roles to checkboxes and fixed validation

### PR DESCRIPTION
**Pull request by vikinatora**
Regarding issue #49 

Additional questions steps:
1. Changed dropdown roles to checkboxes
2. Fixed error messages appearing before submitting the text

Info step: 
1. Fixed validation on GDPR checkbox appearing before submitting

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -35,3 +35,4 @@ coverage
 
 # Env
 .env
+package-lock.json
```
Comment (uzuntonev) [inline .gitignore:38]: I don't think that is a good idea to have two different lockfiles.

**Review by uzuntonev — APPROVED** [review]
Great job! 💯 Perfect error handling. I think that improves user experience. According to me is better to have only one lockfile

File: .gitignore (hunk 1 above)
Comment (kachar) [reply .gitignore:38]: I understood this ignore as a way to disregard the npm lockfile as the lockfile should always be stored in git as we do for `yarn.lock`

Comment (vikinatora) [reply .gitignore]: You are right, removed the line from .gitignore.

**Pull request merged**